### ISSUE TITLE
Vim extension causes error removing non-existent shortcut and Exit and Help buttons not displayed

### ISSUE THREAD

**Issue by bevanj** [damianavila/RISE#269]
When I enable https://github.com/lambdalisue/jupyter-vim-binding I get an error in the js console that leads to line 20845 of main.min.js

>  throw new Error('trying to remove a non-existent shortcut', shortcut, typeof shortcut);

The only consequence that I have seen so far is that the 'RISE exit' and 'Reveal shortcut help' buttons are not displayed on the slideshow.  If I disable this extension, I don't get the error message and the buttons are visible and work as intended.

I am unsure if this was an issue previously.
Thanks

**Comment by damianavila** [comment]
I guess that vim extension is causing some issue that affect RISE down the road. I need to check this setup to see if I can find the relationship.

**Comment by avlahut** [comment]
I believe this is because the f-key is not a shortcut in vim-bindings but the code is trying to remove this shortcut when in reveal mode. The 'exit' and 'help' buttons are placed right below that code so they do not get executed. 

If I place the following as the first cell in a notebook with the vim-bindings plugin, then enter the slideshow, the error goes away and I see the buttons correctly.

`%%javascript
Jupyter.keyboard_manager.command_shortcuts.add_shortcut('f', 'jupyter-notebook:find-and-replace');`

**Comment by damianavila** [comment]
Thanks for the info, @avlahut do you want to make a PR to fix? :wink:

**Comment by avlahut** [comment]
@damianavilla It might take me a bit of time to get the environment set up but I'm up for giving it a shot.

**Comment by damianavila** [comment]
@avlahut sure!!

**Comment by datakurre** [comment]
Might be related that on presentation mode (with vim bindings) arrow up and down do not seem to work (left and right do work).

**Comment by damianavila** [comment]
> arrow up and down do not seem to work

Up and down key should let you browse the cells in the current slide. Are you saying that behavior is disabled?

**Comment by datakurre** [comment]
@damianavila No, it works as expected. Sorry. It seems, I was just confused by https://github.com/damianavila/RISE/issues/180 and though it was caused by a conflict with vim_bindings (it was not). Pgup and Pgdown work fine as you noted in that issue, but I will also welcome new slide navigation as proposed in https://github.com/damianavila/RISE/issues/278 Thanks for fast reponse. I do confirm the original things in this issue.

**Comment by raphaeltimbo** [comment]
> The only consequence that I have seen so far is that the 'RISE exit' and 'Reveal shortcut help' buttons are not displayed on the slideshow. If I disable this extension, I don't get the error message and 

Just to add that Alt + R shortcut is also affected by the vim-binding extension.
